cli/tests/pcluster/api/controllers/test_cluster_instances_controller.py

```diff
@@ -10,6 +10,7 @@
 
 from pcluster.api.models import NodeType
 from pcluster.aws.common import StackNotFoundError
+from pcluster.utils import to_iso_time
```

Comment:
minor: should we try to avoid using `pcluster.utils` in testing?

Reply:
I'm not sure here -- I've generally used converters in the like in testing code to ensure that the logic internally conforms to the conversion operation.

Reply:
Has this come up in the past? Another option is to do the conversions so we have exact expected output.

Reply:
I vaguely remember someone mentioned it. But in this case, I believe it is fine